src/face-fs.glsl

```diff
@@ -18,14 +20,21 @@ vec4 lit(float l ,float h, float m) {
 }
 
 void main() {
-  vec4 texColor = texture2D(u_texture, v_texCoord.xy) * v_color;
   vec3 normal = normalize(v_normal);
   vec3 surfaceToLight = normalize(v_surfaceToLight);
   vec3 surfaceToView = normalize(v_surfaceToView);
-  vec3 halfVector = normalize(surfaceToLight + surfaceToView);
 
-  vec4 litR = lit(dot(normal, surfaceToLight),
-                dot(normal, halfVector), u_shininess);
+  float faceCheck = dot(normal, surfaceToView);
+  float isFront = (sign(faceCheck) + 1.0) * 0.5;
+  float isBack = (-sign(faceCheck) + 1.0) * 0.5;
+  vec3 reflection = vec3(faceCheck, faceCheck, faceCheck);
+
+  vec4 frontTexture = texture2D(u_texture, v_texCoord.xy) * vec4(isFront, isFront, isFront, 1.0);
+  vec4 backTexture = texture2D(u_textureBack, v_texCoord.xy) * vec4(isBack, isBack, isBack, 1.0);
```

Comment:
I'm not sure if this is faster, but avoiding conditionals is great instinct! (No action necessary).